webpack.config.js

```diff
@@ -25,7 +27,14 @@ module.exports = {
       include: path.join(__dirname, 'src')
     }, {
       test: /\.scss$/,
-      loaders: ['style', 'css', 'autoprefixer', 'sass']
+      loaders: ['style', 'css', 'autoprefixer', 'sass?config=sassConfig']
     }]
+  },
+  sassConfig: {
+    functions: {
+      'twemoji-url': function(emoji) {
+        return types.String('url(' + twemoji.base + 'svg/' + twemoji.convert.toCodePoint(emoji.getValue()) + '.svg)');
```

Comment:
This is cool. Had no idea you could "inject" functionality like this. Couple of small nitpicks:

1. Could we do something like `const codePoint = twemoji.convert.toCodePoint(emoji.getValue())`? It just makes the line easier on the :eyes: IMO and shorter.
2. This seems like a good candidate for a template string: `url(${twemoji.base}/svg/${codePoint}.svg)` which are supported natively in recent versions of V8 (plus a ton of other great ES2015 stuff!) :tada:. I think we should keep the rest of the file boring old ES5 because there's not much reason to update, but using ES2015 here would be a  pretty nice cleanup.